pages/refed.tsx

```diff
@@ -17,13 +19,38 @@ function Refed() {
                     </Card.Header>
                     <Card.Body>
                         <p>You were invited to use this application because someone thought it would be interesting and helpful to you.</p>
+                        <p>
+                            After you are logged in with Google you can verify your Minecraft account{' '}
+                            <Tooltip
+                                content={
+                                    <span>
+                                        <HelpIcon />
+                                    </span>
+                                }
+                                type="hover"
+                                tooltipContent={
+                                    <p>
+                                        To connect your Minecraft account, search and click yourself in the search bar. Afterwards click "Your? Claim account."
+                                        to get a full explanation.{' '}
+                                    </p>
+                                }
+                            />{' '}
+                            to get 1 hour of <Link href="/premium">premium</Link> for free to test our service including our{' '}
+                            <Link href="/flipper">auction flipper</Link>.
+                        </p>
+                        <p>
+                            We also provide a mod to use the flipper ingame. You can download it in the <b>#mod-releases</b> channel on our{' '}
+                            <a href="https://discord.gg/wvKXfTgCfb">
+                                <span style={{ color: '#7289da' }}>Discord</span>
+                            </a>
+                            . For help check out the <b>#faq</b> channel or ask in <b>#support</b>.
+                        </p>
                         <hr />
                         <p>Login with Google:</p>
                         <GoogleSignIn onAfterLogin={() => {}} />
                         <p>
-                            We use Google accounts because they are more secure than requiring a separate login. We use your Google Id and email in compliance
-                            with our <a href="https://coflnet.com/privacy">privacy policy</a> (i.e. to know what settings you made and contact you in case we
-                            need to)
+                            We use Google accounts because they are more secure than requiring a separate login. We use your Google Id and email. (i.e. to know
```

Comment:
why was this changed, the new version doesn't make sense. Nobody uses the Google Id or email of our users, we at most store them

Reply:
It is the same text as before, I just removed the part with the privacy policy as it's already mentioned and linked right obove it because of the google login.
![image](https://user-images.githubusercontent.com/58751503/168918097-38a3f556-fd50-4d62-8995-eab514f50271.png)

Reply:
Should the part with the google id and email be removed entirely?

Reply:
```suggestion
                            Settings you make are tied to your google account to sync across devices and into the Minecraft mod. 
```
something like this?
couldn't add the suggestion for more lines since I didn't originally mark them.